cli/tests/pcluster/validators/test_all_validators.py

```diff
@@ -224,6 +224,9 @@ def test_slurm_validators_are_called_with_correct_argument(test_datadir, mocker)
     single_instance_type_subnet_validator = mocker.patch(
         networking_validators + ".SingleInstanceTypeSubnetValidator._validate", return_value=[]
     )
+    enable_single_availability_zone_validator = mocker.patch(
```

Comment:
I think lets test both for true and false (values) for enable_single_availability_zone and different AllocationStrategy